src/components/common/modal.jsx

```diff
@@ -38,7 +38,7 @@ const modalStyles = {
 };
 
 const ModalView = ({serviceName, title, isOpen, closeModal, children}) => (
-    <Modal isOpen={isOpen} onRequestClose={closeModal} style={modalStyles} closeTimeoutMS={200} contentLabel={'Modal'}>
+    <Modal isOpen={isOpen} onRequestClose={closeModal} style={modalStyles} closeTimeoutMS={200} contentLabel={'Modal'} ariaHideApp={false}>
```

Comment:
This resolves the development warning that happens when opening a model, but in the browser and in tests.